source/slang/hlsl.meta.slang

```diff
@@ -14949,6 +14949,16 @@ matrix<T,N,M> WaveMultiPrefixSum(matrix<T,N,M> value, uint4 mask)
     }
 }
 
+[ForceInline]
+[require(hlsl, sm_6_0)]
```

Comment:
The capabilities assigned should be a new capability (add to `source\slang\slang-capabilities.capdef`) roughly equivalent to:
1. sm_6_0 + fragment 
2. GL_EXT_demote_to_helper_invocation + fragment 
3. SPV_EXT_demote_to_helper_invocation + DemoteToHelperInvocationEXT + fragment
3. metal + fragment